src/content/docs/en/recipes/docker.mdx

```diff
@@ -148,10 +148,10 @@ WORKDIR /app
 COPY package.json package-lock.json ./
 
 FROM base AS prod-deps
-RUN npm install --production
+RUN npm install --omit=dev
```

Comment:
Suggestion: Instead of `install`, it would be safer to use `ci`. At this stage, installing is essentially a clean install, but if someone copies this into a different context, they might face some unintended behaviors.

``` suggestion
RUN npm ci--omit=dev
```

Reply:
Question: What should happen with peer and optional dependencies? Should we also omit those, and if so, then perhaps `--include prod` may be better? However, I am not sure that `include` would automatically understand that everything else should be excluded.

Reply:
I agree that ci is better for docker in production for repeatability, but well, for whom that matters this is also easy to modify.

Reply:
I'm going to go with what the error message suggested, and what worked for the OP, and in the spirit of NWTWWHB, `--omit dev` seems appropriate here.